docs/index.rst

```diff
@@ -208,6 +208,39 @@ steps:
          Creating test case test_login_1 for requirement: Login Example.
          Linking test case test_login_1 to requirement: Login Example.
 
+test-plan command
++++++++++++++++++
+
+The test-plan command allows to create either parent or child test plans. This
+is accomplished by passing or not the ``--parent-name`` option, if it is
+present then a child test plan will be created and linked with the specified
+parent and, if not specified, a parent test plan will be created.
+
+The test plan ID and the test plan parent's ID will be generated after their
+names by replacing some special characters and converting spaces to ``_``. You
+don't need to worry about the ID since Betelgeuse will take care of that for
+you, just make sure to pass the exact name for both the test plan and parent
+test plan in order to find the expected work items on Polarion.
+
```

Comment:
How about:

Betelgeuse will automatically generate the test plan IDs from the passed test plan names by replacing special characters and converting spaces to ``_``. 

.. warning::

    Make sure to pass the right names for the test plans in order to find the expected work items in Polarion. Otherwise, you may see an error.